controller/src/controller/main_systems/server.py

```diff
@@ -166,6 +166,11 @@ async def _report_system_error(
         else:
             logger.error("UI has already disconnected, cannot send error message")
 
+    def _get_offline_state(self) -> bool:
+        system_state = self._get_system_state_ro()
+        system_status = system_state["system_status"]
+        return bool(system_status == SystemStatuses.OFFLINE_STATE)
```

Comment:
`system_status == SystemStatuses.OFFLINE_STATE` itself is a bool so `bool()` shouldn't be needed here. If Mypy is complaining about it you can always just use a `type: ignore` comment and some blurb about how Mypy isn't working correctly for this statement